util/src/main/scala/com/scalableminds/util/tools/Fox.scala

```diff
@@ -323,14 +323,24 @@ class Fox[+A](val futureBox: Future[Box[A]])(implicit ec: ExecutionContext) {
       }
     }).flatMap(identity)
 
-  /*
-    Awaits the future and opens the box. Do not use this in production code!
-   */
+  /**
+    *
+    *  Awaits the future and opens the box. Do not use this in production code (therefore marked as Deprecated)!
+    */
+  @Deprecated
   def get(justification: String, awaitTimeout: FiniteDuration = 10 seconds): A = {
-    val box = Await.result(futureBox, awaitTimeout)
+    val box = await(awaitTimeout)
     box.openOrThrowException(justification)
   }
 
+  /**
+    *
+    * Awaits the future and returns the box. Should not be used in production code (therefore marked as Deprecated).
+    */
+  @Deprecated
+  def await(awaitTimeout: FiniteDuration = 10 seconds): Box[A] =
```

Comment:
I think it would be good form to force the user to pass a justification here as well, just like in `get`. This, together with the deprecation warning, hopefully ensures that nobody ever uses this in production code :)